docs/breaking-changes.md

```diff
@@ -14,6 +14,19 @@ This document uses the following convention to categorize breaking changes:
 
 ## Planned Breaking API Changes (37.0)
 
+### Utility Process unhandled rejection behavior change
+
+Utility Processes will now warn with an error message when an unhandled
+rejection occurs instead of crashing the process.
+
+To restore the previous behavior, you can use:
+
+```js
+process.on('unhandledRejection', () => {
+  process.exit(1)
```

Comment:
This is tricky because `process.exit()` does not synchronously crash the utility process. See https://github.com/electron/electron/issues/44174.

I wonder if there's a different function that will kill the process immediately (or if we can get the linked issue fixed for v37).

Reply:
So #45921 above is the PR but it does not have a linked issue attached. It actually looks like you might have been the last one to update the docs for this in https://github.com/electron/electron/pull/46093. Has something changed since then? 

I am all good with updating the docs in addition to the correcting of the release versions. I am just unfamiliar with this code.

Reply:
Chiming in with my two cents: since this is still marked for 37, we have plenty of time to fix the underlying bug (https://github.com/electron/electron/issues/44174) before it goes live. (Which will also require its own breaking change mention anyways lol.)

I'm fine with us leaving the "ideal" code here in the meantime, especially because it's already in this doc.

Reply:
@mlaurencin Yes, I last edited the code fragment to fix a docs mistake in #46093. Unfortunately, I didn't have time to review the code sample when it was first merged.

I'll throw #44174 onto the 37 project board when it is created.